mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -403,10 +400,7 @@ export class ExperimentView extends Component {
           </form>
           <div className="ExperimentView-run-buttons">
             <span className="run-count">
-              {runInfos.length > SEARCH_MAX_RESULTS ?
-                `Showing the latest ${SEARCH_MAX_RESULTS} matching runs` :
-                `${runInfos.length} matching ${runInfos.length === 1 ? 'run' : 'runs'}`
-              }
+              Showing the latest {runInfos.length} matching {runInfos.length === 1 ? 'run' : 'runs'}
```

Comment:
Can we change this to something like 

> Showing ### matching runs

Latest made sense when the UI's `order_by` was `start_time DESC` (and no way to change it)

Reply:
Removed `latest`